src/main/drivers/bus_spi.c

```diff
@@ -17,89 +17,48 @@
 
 #include <stdbool.h>
 #include <stdint.h>
+#include <string.h>
 
 #include <platform.h>
 
+#include "build/debug.h"
+
 #include "drivers/bus_spi.h"
+#include "drivers/bus_spi_impl.h"
```

Comment:
Well, I'd prefer to make that separation in a later PR. Splitting the files in this PR makes it more difficult to review the changes.